internal/gatewayapi/runner/runner.go

```diff
@@ -159,6 +159,7 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 						extGKs = append(extGKs, schema.GroupKind{Group: gvk.Group, Kind: gvk.Kind})
 					}
 					t.ExtensionGroupKinds = extGKs
+					r.Logger.Info("extension resources", "GVKs", extGKs)
```

Comment:
the size of `extGVKs` can be pretty big here, how about recording the length here instead ?